metaseq/modules/activation_functions.py

```diff
@@ -30,6 +30,17 @@ def gelu_back(g, x):
     return ff * g
 
 
+@torch.jit.script
+def relu(x):
+    return F.relu(x)
+
+
+@torch.jit.script
+def relu_back(g, x):
+    g[x <= 0] = 0
```

Comment:
possibly this can add stream synchronization? not really sure, lets just switch to masked_fill_ to be safe?